.github/workflows/test-code.yml

```diff
@@ -0,0 +1,126 @@
+# Tests that rely on the committed code, without needing the docker images to be built
+name: Test EED app
+
+on:
+  push:
+    branches: [develop]
+  pull_request:
+    branches: [develop]
+
+env:
+  PGUSER: postgres
+
+concurrency:
+  group: test-code-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  install-dev-tools:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: dev env setup
+        uses: ./.github/actions/dev-env-setup
+      - run: make install_dev_tools
+      - run: make stop_pg # cleanly stop the database, otherwise it will complain in the subsequent jobs
+      - run: yarn install --frozen-lockfile
+        working-directory: ./app
+
+  trivy-scan-code:
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Run Trivy vulnerability scanner in repo mode
+        uses: aquasecurity/trivy-action@master
+        with:
+          scan-type: fs
+          format: sarif
+          output: trivy-results.sarif
+          exit-code: "0"
+          ignore-unfixed: false
+          severity: UNKNOWN,LOW,MEDIUM,HIGH,CRITICAL
```

Comment:
you might want to increase the trivy timeout to 10 min, see the latest CCBC updates